scipy/stats/tests/test_stats.py

```diff
@@ -3243,6 +3244,19 @@ def test_result_attributes(self):
         attributes = ('statistic', 'pvalue')
         check_named_results(res, attributes)
 
+    @dec.knownfailureif(True, "Tough NIST Test Cases fail")
```

Comment:
There are now 4 test cases without any issues it looks like (``SiRstv``, ``SmLs01``, ``SmLs02``, ``SmLs03``). Those could be in one test case I'd think, which checks to something like ``rtol=1e-7``.

The failing ones vary wildly. They could all be in a single testcase marked as knownfail, but I'm not sure that we expect all of those to pass with ``rtol=1e-7``. So might be the case that some of them are acceptable as is, and could therefore we in a separate testcase with reduced tolerance. The ones that are already way off we don't need to check for any worsening I'd think.